src/config/dynamic-theme-fixes.config

```diff
@@ -2527,6 +2527,23 @@ img[alt*="logo"]
 
 ================================
 
+avnetwork.com
+
+INVERT
+#Group-3
+#Fill-4
+span.site-logo.subsite-logo svg g * *
+span.site-logo.subsite-logo svg path
+#slice-container-subBrandsBar > div > a > span > svg > g:nth-child(1)
+#slice-container-subBrandsBar > div > a > span > svg > path:nth-child(2)
+
+IGNORE INLINE STYLE
+.site-logo svg *
```

Comment:
Let's try solving it a bit differently. Let's try putting something like
```
*
IGNORE INLINE STYLE
svg[role="img"] path
svg[role="img"] g
svg[role="img"] rect
svg[role="img"] circle
// Maybe 'svg[role="img"] *' will work, but not sure about performance
```
and the other fixes should be just `INVERT` when necessary.